src/bp-groups/bp-groups-template.php

```diff
@@ -3392,7 +3395,10 @@ function bp_get_group_reject_invite_link( $group = false ) {
 			$group =& $groups_template->group;
 		}
 
-		$bp = buddypress();
+		$user_domain = bp_loggedin_user_domain();
+		if ( bp_displayed_user_domain() ) {
```

Comment:
I think it's best to use `bp_is_user()` instead of `bp_displayed_user_domain()` here

Reply:
✅